app/src/main/java/io/mochahub/powermeter/data/WorkoutDao.kt

```diff
@@ -0,0 +1,26 @@
+package io.mochahub.powermeter.data
+
+import androidx.lifecycle.LiveData
+import androidx.room.Dao
+import androidx.room.Delete
+import androidx.room.Insert
+import androidx.room.Query
+import androidx.room.Update
+
+@Dao
+interface WorkoutDao {
+    @Query("SELECT * FROM workouts ORDER BY createdAt ASC")
+    fun getAll(): LiveData<List<WorkoutEntity>>
+
+    @Query("SELECT * FROM workouts where workoutSessionUUID= :workoutSessionUUID")
+    fun getWorkoutsByWorkoutSession(workoutSessionUUID: String): List<WorkoutEntity>
```

Comment:
Perfect candidates for returning `Flow`s

Reply:
For `getWorkoutsByWorkoutSession` how would we use Flow for a non LiveData return type? That function is only used in the ViewModel and doesn't need to observe on a LiveData object.

Reply:
https://kotlin.github.io/kotlinx.coroutines/kotlinx-coroutines-core/kotlinx.coroutines.flow/to-list.html

Reply:
:open_mouth: